Assets/HoloToolkit/UX/Scripts/Collections/ObjectCollection.cs

```diff
@@ -62,23 +62,94 @@ public class ObjectCollection : MonoBehaviour
         [Tooltip("Radius for the sphere or cylinder")]
         public float Radius = 2f;
 
+        /// <summary>
+        /// This is the radial range for creating a radial fan layout.
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Radial range for radial layout")]
+        [Range(5f, 360f)]
+        private float radialRange = 180f;
+
+        public float RadialRange
+        {
+            get { return radialRange; }
+            set { radialRange = value; }
+        }
+
         /// <summary>
         /// Number of rows per column, column number is automatically determined
         /// </summary>
+        [SerializeField]
         [Tooltip("Number of rows per column")]
-        public int Rows = 3;
+        private int rows = 3;
+
+        public int Rows
+        {
+            get { return rows; }
+            set { rows = value; }
+        }
 
         /// <summary>
         /// Width of the cell per object in the collection.
         /// </summary>
+        [SerializeField]
         [Tooltip("Width of cell per object")]
-        public float CellWidth = 0.5f;
+        private float cellWidth = 0.5f;
+        public float CellWidth
+        {
+            get { return cellWidth; }
+            set { cellWidth = value; }
+        }
 
         /// <summary>
         /// Height of the cell per object in the collection.
         /// </summary>
+        [SerializeField]
         [Tooltip("Height of cell per object")]
-        public float CellHeight = 0.5f;
+        private float cellHeight = 0.5f;
+
+        public float CellHeight
+        {
+            get { return cellHeight; }
+            set { cellHeight = value; }
+        }
+
+        /// <summary>
+        /// Margin between objects horizontally.
+        /// </summary>
```

Comment:
Summary tags should be above the public properties not the attributes for the private fields